pkgs/development/libraries/audio/lv2/default.nix

```diff
@@ -1,45 +1,73 @@
 { stdenv
 , lib
 , fetchurl
+, asciidoc
+, codespell
+, doxygen
+, gtk2
+, libsamplerate
 , libsndfile
 , pkg-config
 , python3
-, wafHook
+, meson
+, ninja
 , pipewire
+, serd
+, sord
+, gitUpdater
 }:
 
 stdenv.mkDerivation rec {
   pname = "lv2";
-  version = "1.18.2";
+  version = "1.18.10";
 
-  outputs = [ "out" "dev" ];
+  outputs = [ "out" "dev" "doc" ];
 
   src = fetchurl {
-    url = "https://lv2plug.in/spec/${pname}-${version}.tar.bz2";
-    sha256 = "sha256-TokfvHRMBYVb6136gugisUkX3Wbpj4K4Iw29HHqy4F4=";
+    url = "https://lv2plug.in/spec/${pname}-${version}.tar.xz";
+    hash = "sha256-eMUbzyG1Tli7Yymsy7Ta4Dsu15tSD5oB5zS9neUwlT8=";
   };
 
   nativeBuildInputs = [
+    asciidoc
+    codespell
+    doxygen
+    meson
+    ninja
     pkg-config
-    wafHook
-    python3
+    (python3.withPackages (pkgs: with pkgs; [
+      lxml
+      markdown
+      pygments
+      rdflib
+    ]))
+    serd
```

Comment:
AFAICT, this is needed for tests so it might be better to put it into `nativeCheckInputs`.